src/components/Toolbar/VisualizationTypeSelector/VisualizationTypeSelector.js

```diff
@@ -39,12 +45,14 @@ export const VisualizationTypeSelector = ({ visualizationType }) => {
         <Card dataTest={'visualization-type-selector-card'}>
             <div className={classes.listContainer}>
                 <div className={classes.listSection}>
-                    {getVisTypes().map((type) =>
-                        visTypeMap[type].disabled ? (
+                    {[VIS_TYPE_LINE_LIST, VIS_TYPE_PIVOT_TABLE].map((type) =>
```

Comment:
Consider storing two lists, `availableVisTypes` and `disableVisTypes` (or similar names) somewhere central in the app, so we can easily change it in the future (rather than relying on these harcoded checks everywhere)

Reply:
Since we know PT is coming, I thought it's ok to have it like this.
I have to refactor anyway when defining the vis types in `modules/visualization`.